sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -287,6 +287,12 @@ def get_additional_columns(row) -> pd.Series:
             if row['InstanceType'] == 'p4de.24xlarge':
                 acc_name = 'A100-80GB'
                 acc_count = 8
+            if row['InstanceType'] == 'trn1.2xlarge':
```

Comment:
Shall we add a reference link for this instance type? Such as https://aws.amazon.com/ec2/instance-types/trn1/

Reply:
yes, good idea